# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/DeviceInputTrack.cpp
# Commit: a1def130c105
# Full Hash: a1def130c1053111631562df9dbf3545356bc448
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2022-01-28 04:41:16
# Regressor Bug: 1742655
# File Overlap Count: 2
# Description:
#   Bug 1742655 - Remove NotifyOutputData from NativeInputTrack r=pehrsons
#   
#   Depends on D134532
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D134627
# ==============================================================================

diff -r b0a8837110db -r a1def130c105 dom/media/DeviceInputTrack.cpp
--- a/dom/media/DeviceInputTrack.cpp	Thu Jan 27 22:29:34 2022 +0000
+++ b/dom/media/DeviceInputTrack.cpp	Thu Jan 27 22:29:34 2022 +0000
@@ -119,16 +119,6 @@
   return mInputChannels;
 }
 
-void NativeInputTrack::NotifyOutputData(MediaTrackGraphImpl* aGraph,
-                                        AudioDataValue* aBuffer, size_t aFrames,
-                                        TrackRate aRate, uint32_t aChannels) {
-  MOZ_ASSERT(aGraph->OnGraphThreadOrNotRunning());
-  MOZ_ASSERT(aGraph == mGraph, "Receive output data from another graph");
-  for (auto& listener : mDataUsers) {
-    listener->NotifyOutputData(aGraph, aBuffer, aFrames, aRate, aChannels);
-  }
-}
-
 void NativeInputTrack::NotifyInputStopped(MediaTrackGraphImpl* aGraph) {
   MOZ_ASSERT(aGraph->OnGraphThreadOrNotRunning());
   MOZ_ASSERT(aGraph == mGraph,